Firmware/MVP/battsci.cpp

```diff
@@ -117,60 +294,60 @@ void BATTSCI_sendFrames()
           debugUSB_sendChar('7');
-
-        // Regen & Assist, with background charge 
-        } else if (vCellWithESR_counts >= 36000) { //34500 = 3.4500 volts                                            
-          frameSum_87 += BATTSCI_writeByte( 0x14 );                                         //Battery SoC (upper byte)
-          frameSum_87 += BATTSCI_writeByte( 0x58 ); //60% SoC                               //Battery SoC (lower byte)
+        } else if (vCellWithESR_counts >= 34500) { //34500 = 3.4500 volts
           debugUSB_sendChar('6');
-
-        // Regen & Assist, with background charge   
-        } else if (vCellWithESR_counts >= 35000) { //33000 = 3.3000 volts                                              
-          frameSum_87 += BATTSCI_writeByte( 0x13 );                                         //Battery SoC (upper byte)
-          frameSum_87 += BATTSCI_writeByte( 0x10 ); //40% SoC                               //Battery SoC (lower byte)
+        } else if (vCellWithESR_counts >= 33500) { //33500 = 3.3500 volts
           debugUSB_sendChar('4');
-
-        // Battery is empty. Disable Assist.  
         } else {
-          frameSum_87 += BATTSCI_writeByte( 0x11 );                                         //Battery SoC (upper byte)
-          frameSum_87 += BATTSCI_writeByte( 0x48 ); //20% SoC                               //Battery SoC (lower byte)
           debugUSB_sendChar('2');
         }
       }
 
+	// frameSum_87 += BATTSCI_writeByte( 0x21 );                                   //Battery SoC (upper byte)
+	// frameSum_87 += BATTSCI_writeByte( 0x7A );                                   //Battery SoC (lower byte)
+
+	  frameSum_87 += BATTSCI_writeByte( SoC_Bytes[0] );                                   //Battery SoC (upper byte)
+	  frameSum_87 += BATTSCI_writeByte( SoC_Bytes[1] );                                   //Battery SoC (lower byte)
+
       frameSum_87 += BATTSCI_writeByte( highByte(batteryCurrent_toBATTSCI << 1) & 0x7F ); //Battery Current (upper byte)
       frameSum_87 += BATTSCI_writeByte(  lowByte(batteryCurrent_toBATTSCI     ) & 0x7F ); //Battery Current (lower byte)
-      frameSum_87 += BATTSCI_writeByte( 0x32 );                                           //Almost always 0x32
-      
-      int8_t battTempBATTSCI = temperature_battery_getLatest() + 30;                      //T_MCM = T_actual + 30
-      frameSum_87 += BATTSCI_writeByte( battTempBATTSCI );                                //max temp //0 degC = 30d = 0x1E
-      frameSum_87 += BATTSCI_writeByte( battTempBATTSCI );                                //min temp
-      
+      frameSum_87 += BATTSCI_writeByte( 0x32 );                                      //Almost always 0x32 -- 0x00 P1449	0x01 P1449	0x10 P1447	0x11 P1447	0x12 P1447	0X14 P1447	0x15 N	0x16 N	0x17 N	0x20 N	0x30 N	0x31 N	0x33 N	0x7F N
+      frameSum_87 += BATTSCI_writeByte( temperature_Byte );                               //max temp: degC*2 (e.g. 0x3A = 58d = 28 degC
```

Comment:
As mentioned above, I'd rather not change temperature... add a note that we should revisit this once we get a handle on the other flags that can eliminate assist/regen.

Reply:
Changed it so it only modifies temperature to be at least 21ºC, and added note about revisiting once we learn more about the assist and regen flags.